packages/esm-patient-labs-app/src/lab-orders/add-lab-order/test-type-search.component.tsx

```diff
@@ -67,6 +69,7 @@ function TestTypeSearchResults({ searchTerm, openOrderForm, focusAndClearSearchI
   const { t } = useTranslation();
   const isTablet = useLayoutType() === 'tablet';
   const { testTypes, isLoading, error } = useTestTypes();
+  const { orders, setOrders } = useOrderBasket<LabOrderBasketItem>('labs', prepLabOrderPostData);
```

Comment:
Moved the `useOrderBasket` from `TestTypeSearchResults` to its parent. It didn't seem right to have it run for every iteration of the `filteredTestTypes` considering it returns the same value.